best-practices/README.md

```diff
@@ -329,3 +329,18 @@ Angular
 [annotations]: http://robots.thoughtbot.com/avoid-angularjs-dependency-annotation-with-rails
 [ngannotate]: https://github.com/kikonen/ngannotate-rails
 [angular-translate]: https://github.com/angular-translate/angular-translate/wiki/Getting-Started#using-translate-directive
+
+Ruby JSON APIs
+--------------
+
+* Review the recommended practices outlined in Heroku's [HTTP API Design Guide]
+  before designing a new API.
+* Use a fast JSON parser, e.g. [`oj`][oj]
```

Comment:
Oj out of the box doesn't do exactly what you might expect:

```
irb(main):004:0> puts Oj.dump(foo: 'baz')
{":foo":"baz"}
```

I'd add a note here that for Ruby hashes w/ symbols you need to enable `compat` mode.